conftest.py

```
@@ -1,320 +1,253 @@
+    result = "unknown"
+
+    try:
+        # Run the test
+        yield
+        result = "passed"
+
+    except KeyboardInterrupt:
+        result = "interrupted"
+        raise
+
+    except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)